src/components/AudioPlayer/AudioPlayer.module.scss

```diff
@@ -2,6 +2,10 @@
 @use "src/styles/constants";
 @use "src/styles/utility";
 
+.emptySpacePlaceholder {
```

Comment:
Might be good to add a comment here explaining why we have this placeholder

Reply:
nit: I would call this heightPlaceholder. but not a strong opinion. up to you :)